tests/test_store.py

```diff
@@ -3,15 +3,20 @@
 import unittest
 
 import mock
+from openpyxl.conftest import Workbook
```

Comment:
Imports should be ordered in blocks for standard library, external dependencies, internal files